feature_server/run.py

```diff
@@ -57,6 +57,10 @@ def choose_path(base,top):
 
 # add it to the path so we can import scripts
 sys.path.append(config_dir)
+# add our package to path too so scripts can import `feauture_server/`
```

Comment:
Yes, but it would be accessible as `import pysnip; print pysnip.commands` (pysnip=feature_server, see line https://github.com/feikname/spades-server/pull/14/files#diff-2eeaed663bd0d25b7e608891384b7298R49). The way I'm doing it there allows scripts to access it like this: `import commands`